C/csb/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "csb"
```

Comment:
The name seems to be spelled all capital (CSB):
```suggestion
name = "CSB"
```
You'll also have to rename the directory accordingly.